techniques/system/rudderSlapd/1.0/main.st

```diff
@@ -26,8 +26,9 @@ bundle common rudder_slapd {
 bundle agent system_rudder_slapd_configuration {
   vars:
 
-      "slapd_password" string => "&RUDDER_LDAP_PASSWORD&";
-
+      #"slapd_password"   string => "&RUDDER_LDAP_PASSWORD&";
```

Comment:
We can not yet put dynamic informations such as LDAP password in the directive parameters in the current state of Rudder. So I fallback on the old way: parsing the rudder-passwords.conf